modules/ROOT/pages/custom-application-alerts.adoc

```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
+so that they can fix the underlying problems.
 
-* Your application must be deployed to CloudHub. +
-xref:deploying-to-cloudhub.adoc[Deploy to CloudHub]
-* You Mule project must be mavenized. +
-** If you are using Studio 6.x and earlier, make sure you xref:6@studio::enabling-maven-support-for-a-studio-project.adoc[Mavenize your project in Studio].
-** All projects in Studio 7.x and later are mavenized by default.
 
+[NOTE]
+Custom alerts and notifications are available only for apps deployed to CloudHub workers.
+They are not available for apps deployed to on-premises servers.
+See xref:deployment-strategies.adoc[Deployment Options] for more information.
 
-== How to Use Notifications
 
-Notifications give you visibility into business-related events inside your application. For example, you can create notifications for when:
+== Prerequisites
 
-* Your application is unable to connect to a remote service.
-* An error that requires human intervention occurs, such as a problem with data mapping.
-* You want to create a summary of what occurred inside your application, such as the number of orders processed.
+* Deploy your application to CloudHub.
++
+See xref:deploying-to-cloudhub.adoc[Deploy to CloudHub]
+* For Anypoint Studio 6.x and earlier, enable Maven support for your Mule project.
++
+See xref:6@studio::enabling-maven-support-for-a-studio-project.adoc[Enabling Maven Support for a Studio Project].
++
+All projects in Studio 7.x and later are mavenized by default.
 
-In combination with xref:alerts-on-runtime-manager.adoc[alerts], you can direct these notifications to different people to act on the alerts and fix the underlying problems.
 
-This example illustrates:
 
-* How to send business related events as notifications
-* How to send errors as notifications
-* How to send email alerts when notifications arrive
+== Custom Application Alerts
 
-[NOTE]
-CloudHub stores up to a maximum of 1000 notifications per application.
-If the number of notifications exceeds the maximum,
-CloudHub deletes the oldest notification.
-Each notification can contain up to 10,000 characters.
+Alerts enable you to send emails based on your application's notifications.
+You can create standard alerts in Runtime Manager for servers and applications deployed to either CloudHub or on-premises servers.
+See xref:alerts-on-runtime-manager.adoc[Alerts].
+
+
+You can also create _custom alerts_ for applications deployed to CloudHub.
+To create custom application alerts, you must add Anypoint Connector for CloudHub (CloudHub Connector)
+to flows in your Mule application.
+CloudHub Connector is available for both Mule 3 and Mule 4 applications.
 
-== Deploy Your Application
+=== Create a Custom Alert
 
-To deploy your application, in the Runtime Manager click *Deploy Application* from the *Applications* menu. For more information, see xref:deploying-to-cloudhub.adoc[Deploy to CloudHub].
+To create a custom alert in Anypoint Studio:
 
```

Comment:
I wonder if the first three steps below aren't actually a separate, prerequisite topic to the actual creation of the alert, beginning with "binding the custom notification to an alert"?